src/server/scripts/Shadowlands/Sanctum of Domination/boss_sylvanas_windrunner.cpp

```diff
@@ -0,0 +1,3691 @@
+    SPELL_JAILER_CAST_SOULS                     = 355348,
+
+    // Anduin Wrynn
+    SPELL_FOCUSING_PRISM_AURA                   = 358166, // Related to the achievement
+
+    SPELL_INTERMISSION_SCENE                    = 359062,
+
+    SPELL_FINAL_SCENE                           = 358806,
+    SPELL_FINAL_CINEMATIC                       = 358985,
+    SPELL_SYLVANAS_FEIGN_DEATH                  = 358844,
+
+    SPELL_HEALTH_CHECK_INTERMISSION             = 358794, // These spells check Sylvanas' health pool to act like phase switchers
+    SPELL_HEALTH_CHECK_LAST_PHASE               = 359430
+};
+
+enum Phases
+{
+    PHASE_ONE                                   = 1,
+    PHASE_INTERMISSION                          = 2,
+    PHASE_TWO                                   = 3,
+    PHASE_THREE                                 = 4
+};
+
+enum Events
+{
+    EVENT_WINDRUNNER                            = 1,
+    EVENT_WITHERING_FIRE,
+    EVENT_SHADOW_DAGGERS,
+    EVENT_DESECRATING_SHOT,
+    EVENT_DESECRATING_SHOT_LAUNCH,
+    EVENT_DOMINATION_CHAINS,
+    EVENT_WAILING_ARROW,
+    EVENT_VEIL_OF_DARKNESS,
+    EVENT_RANGER_HEARTSEEKER,
+    EVENT_RIVE,
+    EVENT_FINISH_INTERMISSION,
+
+    EVENT_DESECRATING_SHOT_SPIRAL
+};
+
+enum Actions
+{
+    ACTION_REGENERATE_POWER                     = 2,
+    ACTION_PAUSE_AI_FOR_EVENT,
+    ACTION_REACTIVATE_AI_AFTER_EVENT,
+    ACTION_WINDRUNNER_MODEL,
+    ACTION_DESECRATING_SHOT_LAUNCH,
+    ACTION_ACTIVATE_DOMINATION_ARROW,
+    ACTION_WAILING_ARROW,
+    ACTION_PREPARE_PHASE_THREE,
+    ACTION_OPEN_PORTAL_TO_PHASE_THREE,
+    ACTION_INITIATE_PHASE_THREE,
+    ACTION_START_PHASE_THREE
+};
+
+enum Texts
+{
+    SAY_AGGRO                                   = 0,
+    SAY_SLAY                                    = 1,
+    SAY_DISENGAGE                               = 2,
+    SAY_ANNOUNCE_WINDRUNNER                     = 3,
+    SAY_SHADOW_DAGGER                           = 4,
+    SAY_DESECRATING_SHOT                        = 5,
+    SAY_ANNOUNCE_WAILING_ARROW                  = 6,
+    SAY_ANNOUNCE_WAILING_ARROW_TANK             = 7,
+    SAY_WAILING_ARROW                           = 8,
+    SAY_ANNOUNCE_DOMINATION_CHAINS              = 9,
+    SAY_DOMINATION_CHAINS                       = 10,
+    SAY_ANNOUNCE_VEIL_OF_DARKNESS               = 11,
+    SAY_VEIL_OF_DARKNESS                        = 12,
+    SAY_INTERMISSION_BEGIN                      = 13,
+    SAY_INTERMISSION_END                        = 14,
+
+    SAY_START_PHASE_THREE                       = 25
+};
+
+enum Conversations
+{
+    CONVERSATION_INTRO                          = 17368
+};
+
+enum Points
+{
+    POINT_INTRO_01                              = 1,
+    POINT_INTRO_02                              = 2,
+    POINT_INTRO_03                              = 3,
+    POINT_INTRO_FINISHED                        = 4
+};
+
+enum VisualKits
+{
+    VISUAL_KIT_SYLVANAS_INTRODUCTION_TALK_01      = 150067,
```

Comment:
inconsistent naming.. for VISUALS its SPELL_VISUAL_XXX, here its VISUAL_KIT_SYLVANAS_XXX

Reply:
fixed on next commit.